reconcile/saas_auto_promotions_manager/publisher.py

```diff
@@ -39,7 +39,7 @@ def __init__(
         target_name: str | None,
         cluster_name: str,
         auth_code: HasSecret | None,
-        publish_job_logs: bool | None,
+        re_deploy_on_config_change: bool | None,
```

Comment:
nit: since redeploy is actually a word: https://www.merriam-webster.com/dictionary/redeploy

```suggestion
        redeploy_on_config_change: bool | None,
```

Reply:
Good point. In that case I should adjust the naming in schema too. On it :+1:

Reply:
Fixed everywhere